### PR TITLE
Local network tutorials refactor

### PR DESCRIPTION
**Pull request by ElPaisano**
This PR proposes:
- Consolidating @rjan90 's [local network with Fil+ PR](https://github.com/filecoin-project/lotus-docs/pull/374 ) and t[he existing local network tutorial](https://lotus.filecoin.io/lotus/developers/local-network/) into one page 
-  Deduplicating or linking to the following, as both tutorials share many prereq, set up and start steps:
   - Minimum system requirements
   - System specific prerequisites
   - (Most) Lotus node set up steps
   - (Most) steps for starting the Lotus node
- Marking steps and actions that are specific to the Fil+ set up as such, and letting readers know they can skip that step IF they are just setting up a local net w/o Fil+

Next steps:
- Need to make some more smallish edits to grammar, syntax, structure
- test the guide one more time, testing with and without Fil+ set up
- check with reviewers 

Benefits:
- Cuts down on a ton of duplicate content  (Minimum system requirements, System specific prerequisites, most Lotus node set up steps, many steps for starting the Lotus node
- By consolidating, we go from two tutorials with about 800 lines between them, to one guide with about half as many lines
- As we add more local network specific content, we can just add in new blurbs and sections to one page, as opposed to adding new pages, etc.

### CODE REVIEW

**Comment by johnnymatthews** [comment]
This is a great first attempt @ElPaisano! I like the idea of merging everything together like this; makes things a bit easier to maintain. 

I've submitted a few change to formatting, and Hugo quirks. Here's a summary:

- Use `_` for emphasis.
- Use `{{< alert >}}` shortcode for callouts.
- For order lists, always use `1.` at the start.
- Always attach a syntax to codeblocks. If there isn't an appropriate syntax, use plaintext: ```plaintext
- Codeblocks should be surrounded by empty lines (see point below).
- To indicate the output of a command, put the input in one codeblock, and then the output in another codeblock immedietly following the first:

    ````markdown
    1. Lorem ipsum:
        
        ```shell
        cat hello.txt
        ```

        ```
        Hello world
        ```

    1. Lorem ipsome more.
    ````

1. Codeblocks must be indented by 4 spaces when under a list item (see above point).

It looks like the section from line 313 down isn't complete yet. Ping me again when you're up for another review.

**Comment by ElPaisano** [comment]
update on this:

@rjan90 this tutorial is basically done. However, per our discussion, I need assistance troubleshooting the following before we can call it done:

I'm still stuck on the `lotus msig inspect f080` step in the Add Notaries section. The error is `ERROR: unknown actor code bafk2bzaced4gcxjwy6garxwfw6y5a2k4jewj4t5nzopjy4qwnimhjtnsgo3ss`. I'm running `lotus version 1.17.2+mainnet+git.fb0fb7144`. The ` lotus state actor-cids` command shows that `multisig`  is `bafk2bzaced4gcxjwy6garxwfw6y5a2k4jewj4t5nzopjy4qwnimhjtnsgo3ss`. 

From our discussion, it sounds like it is related to the ordering of the steps 6-8 in the "Set up a Lotus Node" section. Currently, I have them ordered as Recursively update submodules -> build any new submodules -> build lotus based on your original PR. Additionally, you mentioned there might be an issue with step where I build any new submodules, as it's using a random commit.

So, three questions:
1. How do I troubleshoot the `lotus msig inspect f080` step in the Add Notaries section?
2. How should steps 6-8 in the "Set up a Lotus Node" section be ordered?
3. In the build submodules step, what commit should be used? 

@johnnymatthews I've gone through a few times and (hopefully) caught any of my formatting issues, etc.. Just need to figure out the stuff above and then I think this will be ready to merge. Tried to clean up my wording using vale linter and checked links using markdown-link-check. LMK if you see any other formatting issues, etc..

Also, do you guys like the troubleshooting section?

**Comment by ElPaisano** [comment]
@jennijuju the issue(s) I'm stuck on are described above

**Comment by johnnymatthews** [comment]
Submitted a couple of minor grammar and syntax fixes -- mostly stuff like tab width and spacing.

Regarding the troubleshooting section: [we have a troubleshooting section within `/reference/basics`](https://lotus.filecoin.io/reference/basics/troubleshooting/). @rjan90, what do you think about creating another troubleshooting page within this section? Or would you rather keep troubleshooting steps to the page where someone can encounter them? The benefit of the former is that it'll probably improve our SEO, should someone simply Google their error message.

**Comment by rjan90** [comment]
Hey! I owe some follow-ups on this PR.

> How do I troubleshoot the lotus msig inspect f080 step in the Add Notaries section? ERROR: unknown actor code bafk2bzaced4gcxjwy6garxwfw6y5a2k4jewj4t5nzopjy4qwnimhjtnsgo3ss

So this issue stems from the the submodules that where updated and rebuilt from a wrong commit (that was in my draft) that where not related to the stable release that was checked out. To try to fix this on your local-net, you can try to stop everything -> `git submodule update` in your lotus-local-net folder, `make clean` and `make 2k` again.

> How should steps 6-8 in the "Set up a Lotus Node" section be ordered?
> In the build submodules step, what commit should be used?

I would just remove step `7. recursively update submodules` and `8. Build any new submodules` entirely. Those should already be up to date when cloning the most recent lotus-repo, checking out the latest releases and building the dev-net.

**Comment by rjan90** [comment]
> Regarding the troubleshooting section: [we have a troubleshooting section within /reference/basics](https://lotus.filecoin.io/reference/basics/troubleshooting/). @rjan90, what do you think about creating another troubleshooting page within this section? Or would you rather keep troubleshooting steps to the page where someone can encounter them? The benefit of the former is that it'll probably improve our SEO, should someone simply Google their error message.

I do not have any strong opinions on this, but I´m a bit hesitant to add another troubleshooting section. It would be nice to align on this during LabWeek. The Lotus-docs currently has multiple places that list some sort of troubleshooting help. Either in page, [troubleshooting section within /reference/basics](https://lotus.filecoin.io/reference/basics/troubleshooting/), and the [knowledge base](https://lotus.filecoin.io/kb/).

**Comment by johnnymatthews** [comment]
> the [knowledge base](https://lotus.filecoin.io/kb/).

I FORGOT ABOUT THE KNOWLEDGE BASE! 😅 We can probably move troubleshooting stuff into there, and tag it properly so that it gets good SEO.

> align on this during LabWeek.

Agreed.

**Comment by ElPaisano** [comment]
howdy, I have remove the steps discussed above and the troubleshooting section (keeping those notes so we can add them to the knowledge base at a later time if that's the consensus)

**Comment by ElPaisano** [comment]
Hey @johnnymatthews I can't merge this.

File: content/en/lotus/developers/local-network.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 title: "Local network"
-description: "Running a Filecoin network locally can be extremely useful for developers wanting to build and test their applications. This page provides guidance on different methods to run a Filecoin network locally."
-lead: "Running a Filecoin network locally can be extremely useful for developers wanting to build and test their applications. This page provides guidance on different methods to run a Filecoin network locally."
+description: "This tutorial describes how set up and run a local Filecoin network (local-net) with 2 KiB sectors and root key holders using the regular Lotus binaries. A local-net is useful for developing applications and testing features, like Filecoin+ (Fil+)."
+lead: "This section tutorial how set up and run a local Filecoin network (local-net) with 2 KiB sectors and root key holders using the regular Lotus binaries. A local-net is useful for developing applications and testing features, like Filecoin+ (Fil+)."
```
Comment (rjan90) [inline content/en/lotus/developers/local-network.md:4]: Small typo here, should probably be `This section tutorial describes how to set up`

**Pull request merged**